Formula/gitleaks.rb

```diff
@@ -0,0 +1,24 @@
+class Gitleaks < Formula
+  desc "Audit git repos for secrets"
+  homepage "https://github.com/zricethezav/gitleaks"
+  url "https://github.com/zricethezav/gitleaks/archive/v1.24.0.tar.gz"
+  sha256 "6ba812be47976ca49bc2f5ab888c44ef41b824dd20fa9be5687f4ff6d185c2b1"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = bin
+    dir = buildpath/"src/github.com/zricethezav/gitleaks"
+    dir.install buildpath.children
+    cd dir do
+      system "dep", "ensure", "-vendor-only"
+      system "go", "install"
+    end
+  end
+
+  test do
+    system "#{bin}/gitleaks", "-v"
```

Comment:
Could we get a test that cover more functionalities of `gitleaks`? The Readme of `gitleaks` shows this example:
```
gitleaks --repo=https://github.com/gitleakstest/gronit --verbose
```
You could then assert on some part of the output.

Reply:
@Moisan good call, but the tests are running into timeouts since gitleaks takes a second to clone the remote repos. Anyway to increase the timeout duration?

Reply:
Looks like empty repos could make it under the timeout duration. I tried testing it on another repo I just created https://github.com/gitleakstest/dummyrepo with 2 commits but that didn't make the cut either.

Reply:
@Moisan Lemme know if that works for ya

Reply:
Yes that's great! Thanks!